unixfs/README.md

```diff
@@ -1,32 +1,15 @@
 ![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square) unixfs
 ================================================================================
 
-Authors:
+Draft work and discussion on the upcoming UnixFS specification is happening in the [`ipfs/ipld-unixfs` repo](https://github.com/ipfs/unixfs-v2). Please see the issues there for discussion and PRs for drafts. When the specification is completed there, it will be copied back to this repo and replace this document.
```

Comment:
@Mr0grog the repo name was updated to a more accurate name -- https://github.com/ipfs/unixfs-v2/.

AFAIU, https://github.com/ipfs/unixfs-v2/ is about exploring the same to design the next unixfs and not spec the current implementation.